.github/actions/send-telegram-notify/action.yml

```diff
@@ -72,8 +72,8 @@ runs:
         fi
         # Use MarkdownV2 while Markdown is legacy
         # https://core.telegram.org/bots/api#markdownv2-style
-        python -c "import urllib, requests ; \\
+        python3 -c "import urllib, requests ; \\
```

Comment:
Sorry, this comment does not rely on your changes, but I'm interesting in why we use here both `urrlib` and `requests` if module `urllib` provides `POST` method.

Reply:
Right, better to use only urllib. Patch corrected.